src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -575,6 +594,10 @@ private static bool TryParseStatusLine(HttpResponseMessage response, string resp
                         int reasonPhraseEnd = newLineCharIndex >= 0 ? newLineCharIndex : responseHeaderLength;
                         response.ReasonPhrase = responseHeader.Substring(reasonPhraseIndex, reasonPhraseEnd - reasonPhraseIndex);
                     }
+                    state._isRedirect = state._handler.AutomaticRedirection &&
+                         (response.StatusCode == HttpStatusCode.Redirect ||
+                         response.StatusCode == HttpStatusCode.RedirectKeepVerb ||
+                         response.StatusCode == HttpStatusCode.RedirectMethod) ;
```

Comment:
Yes. But the  second response could be a further redirect. So there are cases where, when EasyRequest reaches a finalization, _isRedirect is still true. Do you see any problem with this?